boards/xtensa/intel_s1000_crb/support/device.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Author: Sathish Kuttan <sathish.k.kuttan@intel.com>
+
```

Comment:
Would be good to include some documentation comments here about what this script is for (same goes for the other Python scripts in this PR).